src/form/Select/Select.tsx

```diff
@@ -84,6 +84,11 @@ export interface SelectProps {
    */
   multiple?: boolean;
 
+  /**
+   * The display type of the selected values.
+   */
+  multipleDisplayType?: 'chip' | 'count';
```

Comment:
What is this count mode?

Reply:
It's mode where displayed counter of selected items instead chips
![Screenshot 2024-05-27 at 17 40 08](https://github.com/reaviz/reablocks/assets/5399389/3cd07142-9825-46ee-8ff9-e9bbe0194997)